.github/workflows/scripts/qemu-3-deps.sh

```diff
@@ -30,11 +30,11 @@ function debian() {
 
   echo "##[group]Install Development Tools"
   sudo apt-get install -y \
-    acl alien attr autoconf bc cpio curl dbench dh-python dkms fakeroot \
-    fio gdb gdebi git ksh lcov isc-dhcp-client jq libacl1-dev libaio-dev \
-    libattr1-dev libblkid-dev libcurl4-openssl-dev libdevmapper-dev libelf-dev \
-    libffi-dev libmount-dev libpam0g-dev libselinux-dev libssl-dev libtool \
-    libtool-bin libudev-dev libunwind-dev linux-headers-$(uname -r) \
+    acl alien attr autoconf bc cpio cryptsetup curl dbench dh-python dkms \
+    fakeroot fio gdb gdebi git ksh lcov isc-dhcp-client jq libacl1-dev \
+    libaio-dev libattr1-dev libblkid-dev libcurl4-openssl-dev libdevmapper-dev \
+    libelf-dev libffi-dev libmount-dev libpam0g-dev libselinux-dev libssl-dev \
+    libtool libtool-bin libudev-dev libunwind-dev linux-headers-$(uname -r) \
```

Comment:
It looks like adding `cryptsetup` breaks causing the dependency install to wait on Debian 11.

https://github.com/openzfs/zfs/actions/runs/11484989922/job/31985681440?pr=16681

Reply:
I opened a fix for it: https://github.com/openzfs/zfs/pull/16683